frontend/src/App.jsx

```diff
@@ -1,37 +1,33 @@
-import { BrowserRouter, Route, Routes } from 'react-router-dom'
-import { useEffect, useState } from 'react'
-import HomePage from './pages/HomePage'
-import NFTMarketplace from './components/NFTMarketplace'
-import ExplorePage from './pages/ExplorePage'
-import AboutPage from './pages/AboutPage'
-import CreatorsPage from './pages/CreatorsPage'
-import FAQPage from './pages/FAQPage'
-import Navbar from './components/Navbar'
-import CreateNFT from './pages/CreateNFTPage'
-import NotFoundPage from './components/NotFoundPage'
-import Footer from './components/Footer'
-import CustomCursor from './components/CustomCursor'
```

Comment:
you've not added custom cursor component,

Reply:
app.jsx is updated pls check